app/forms/sign_in_or_continue_form.rb

```diff
@@ -0,0 +1,43 @@
+class SignInOrContinueForm < Form
+  # - Sign in with Teacher ID does NOT save or interact with this Form and POSTs straight to Teacher ID
+  # - If Teacher ID is disabled for the journey, we still mimick if a user clicked on "Continue without signing in".
+
+  def initialize(claim:, journey:, params:)
+    super
+
+    skip_form_if_teacher_id_not_enabled_for_journey
+  end
+
+  def save
+    continue_without_teacher_id
+
+    true
+  end
+
+  # This is needed because logging in with Teacher ID doesn't submit to this form.
+  # Therefore the slug/page needs to be considered submitted to avoid it being the next slug
+  # after callback redirect back to Claim from Teacher ID.
```

Comment:
What happens if we remove the `sign-in-or-continue` slug from the sequence and start the journey at `teacher-details`? We'd need to probably need to point the "sign-in-or-continue" form at it's own controller to handle some stuff around reseting details from DfE identify when skip teacher id is true but it might help us simplify the claims controller. Probably too much effort at the minute just an idea!